4_Database_setup.md

```diff
@@ -1,26 +1,31 @@
 ---
 layout: default
-title: Database setup
+title: Data beyond a CSV
 ---
 
-# Database basics
+# Data beyond a CSV
 
 > Nelisa needs a more effective way of storing weekly sales data and profits.
 
 Create a database for Nelisa to store Products, Categories, Sales and Purchases.
 
+Add Categories to the database using a [SQL script](http://database.projectcodex.co/steps/running_sql_scripts.html).
+
+Import weekly sales & purchases files into [MySQL using NodeJS](https://www.npmjs.com/package/mysql), using [bulk insert](http://database.projectcodex.co/steps/mysql_bulk_insert_using_nodejs).
+
 Add support for showing the Products, Categories, Sales and Purchases from the database in a web browser.
 
-Import weekly sales and purchases files into the database.
+Learn more about databases [here](http://database.projectcodex.co)
 
 # Learning areas
```

Comment:
Can we be consistent and use present continuous tense? setting up MySQL, modelling your first db.